test/test_coordgen.cpp

```diff
@@ -450,4 +450,32 @@ BOOST_AUTO_TEST_CASE(testClockwiseOrderedSubstituents)
     BOOST_REQUIRE_EQUAL(orderedNeighbors[0], neigh1);
     BOOST_REQUIRE_EQUAL(orderedNeighbors[1], neigh3);
     BOOST_REQUIRE_EQUAL(orderedNeighbors[2], neigh2);
-}
\ No newline at end of file
+}
+
+BOOST_AUTO_TEST_CASE(testbicyclopentane)
+{
+    /*
+     test that bicyclo(1,1,1)pentane is rendered without clashes between the bridge carbons
+     CRDGEN-270
+     */
+
+    auto mol = "C1C2CC1C2"_smiles;
+    sketcherMinimizer minimizer;
+    minimizer.initialize(mol); // minimizer takes ownership of mol
+    minimizer.runGenerateCoordinates();
+
+    const auto& atoms = minimizer._molecules[0]->getAtoms();
+    auto bridgeAtom1 = atoms.at(1);
+    auto bridgeAtom2 = atoms.at(2);
+    auto bridgeAtom3 = atoms.at(3);
+    BOOST_TEST(bridgeAtom1->neighbors.size() == 2);
+    BOOST_TEST(bridgeAtom2->neighbors.size() == 2);
+    BOOST_TEST(bridgeAtom3->neighbors.size() == 2);
+    auto distance1 = (bridgeAtom1->getCoordinates() - bridgeAtom2->getCoordinates()).length();
```

Comment:
Would this be a good test case for `areBondsNearIdeal()`?

Reply:
no, these atoms are not bonded together and the distance is different from a bond lengths. This is testing that they are not too close to each other and drawn on top of each other